x-pack/plugins/apm/public/components/app/metrics/static_dashboard/dashboards/nodejs.json

```diff
@@ -0,0 +1 @@
:0},\"gridlinesVisibilitySettings\":{\"x\":true,\"yLeft\":true,\"yRight\":true},\"preferredSeriesType\":\"area\",\"layers\":[{\"layerId\":\"0342e75d-b1e9-4b7b-b919-91d00be8347a\",\"accessors\":[\"5fe6a2d4-0bc8-4d7d-8b5d-11dfb414a3e3\"],\"position\":\"top\",\"seriesType\":\"area\",\"showGridlines\":false,\"layerType\":\"data\",\"xAccessor\":\"2df8da37-5823-488d-bbb7-9cbd0cd8f270\"}]},\"query\":{\"query\":\"\",\"language\":\"kuery\"},\"filters\":[],\"datasourceStates\":{\"formBased\":{\"layers\":{\"0342e75d-b1e9-4b7b-b919-91d00be8347a\":{\"columns\":{\"2df8da37-5823-488d-bbb7-9cbd0cd8f270\":{\"label\":\"@timestamp\",\"dataType\":\"date\",\"operationType\":\"date_histogram\",\"sourceField\":\"@timestamp\",\"isBucketed\":true,\"scale\":\"interval\",\"params\":{\"interval\":\"auto\",\"includeEmptyRows\":true,\"dropPartials\":false}},\"5fe6a2d4-0bc8-4d7d-8b5d-11dfb414a3e3\":{\"label\":\"Avg. Delay [ms]\",\"dataType\":\"number\",\"operationType\":\"average\",\"sourceField\":\"nodejs.eventloop.delay.avg.ms\",\"isBucketed\":false,\"scale\":\"ratio\",\"params\":{\"emptyAsNull\":true,\"format\":{\"id\":\"number\",\"params\":{\"decimals\":2}}},\"customLabel\":true}},\"columnOrder\":[\"2df8da37-5823-488d-bbb7-9cbd0cd8f270\",\"5fe6a2d4-0bc8-4d7d-8b5d-11dfb414a3e3\"],\"incompleteColumns\":{},\"sampling\":1}}},\"textBased\":{\"layers\":{}}},\"internalReferences\":[],\"adHocDataViews\":{}}},\"hidePanelTitles\":false,\"enhancements\":{}},\"title\":\"Average event loop delays\"}]","timeRestore":false,"title":"Node.js - Runtime Metrics","version":1},"coreMigrationVersion":"8.8.0","created_at":"2023-05-15T12:58:15.826Z","id":"4e597bf0-f2f5-11ed-86ef-2d34c38162bc","managed":false,"references":[{"id":"APM_STATIC_DATA_VIEW_ID","name":"1dfe1c75-8222-4a2f-b441-e548df073c86:indexpattern-datasource-layer-e80a135f-3804-46aa-9bd1-ab2293865d33","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"d4f73ccb-9a7d-4807-ba86-ebe99fd238f2:indexpattern-datasource-layer-4583ccb0-ec13-4715-828e-4800b9c82638","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"30774e51-811d-45f8-89b3-40aaaed13d5c:indexpattern-datasource-layer-f80ca0c4-a851-4e30-bd49-15d94c0a3c02","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"aa8f2bb2-1afd-4e53-8b4c-de6304681854:indexpattern-datasource-layer-c9ffe758-50f3-42d4-ad6c-e77d45d824e1","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"046dceee-2842-4798-8aa4-d41250b0e0b5:indexpattern-datasource-layer-4ccf7a4f-d253-4c0c-b97d-b0a27cd823c8","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"5358d63e-beeb-4e94-9aeb-03cd5a3992ac:indexpattern-datasource-layer-28581c0a-210b-45cf-b373-be97cf61f556","type":"index-pattern"},{"id":"APM_STATIC_DATA_VIEW_ID","name":"6691fa3a-2a57-442a-969a-50a69756eeea:indexpattern-datasource-layer-0342e75d-b1e9-4b7b-b919-91d00be8347a","type":"index-pattern"}],"type":"dashboard","typeMigrationVersion":"8.7.0","updated_at":"2023-05-15T12:58:15.826Z","version":"WzEwNzM5LDNd"}
```

Comment:
What's the purpose of this file? I see it has a hardcoded `service.name: frontend-node` which I don't understand.

Reply:
> What's the purpose of this file?

That's the exported dashboard meant to be shown for all services with `agent.name = nodejs`. Here, it's basically just an example to test with.

> I see it has a hardcoded service.name: frontend-node which I don't understand.

That's a bug, will fix that.